### ISSUE TITLE
Integrate DF in the configuration

### ISSUE THREAD

**Issue by SergeStinckwich** [PolyMathOrg/PolyMath#42]
PolyMath should load DF : https://github.com/PolyMathOrg/DataFrame

**Comment by jecisc** [comment]
Do you want to point to a branch or a release?

**Comment by SergeStinckwich** [comment]
I guess master branch. @olekscode do you have a stable version ?

**Comment by jecisc** [comment]
Other question: What do you want to load in PolyMath? All the project or a group?

Because I see for example Roassal in the Tools dependency, and it's a pretty big project. So I was wondering if you want to load the 'Core' group or the 'default' group?

**Comment by olekscode** [comment]
I don't think we should load `DataFrame-Tools` because they depend on many other packages such as Roassal. `DataFrame-Core` is completely independent. Then Tools can be loaded separately. I'm not sure how to organize this, but the idea is the following:

1. You load PolyMath and get DataFrame as a lightweight data structure
2. If you want additional tools like visualizations, database management, you can load them later as a plug-in.

**Comment by olekscode** [comment]
About the stable branch. DataFrame's master branch should have a stable version, but I will double-check it. However, I would like to change some things before ESUG. So it's stable but has to be updated.

**Comment by olekscode** [comment]
Another question is whether we really want to add DataFrame inside PolyMath.

`scikit-learn` and `scipy` work with `pandas`, but they don't come with `pandas` pre-installed. I think we need to understand how closely is DataFrame related to the rest of PolyMath and should we really load it with the same script.

If algorithms of PolyMath (such as linear regression or genetic algorithms) were implemented on top of DataFrame - then yes. But at this moment everything is based on `PMVector` and `PMMatrix`. And I think that it's good. DataFrame is a specialized data structure and perhaps most people who will use PolyMath for matrix algebra or numeric integration don't need DataFrame.

So yes, PolyMath definitely needs a data structure like DataFrame, but I think that it should be loaded with a separate configuration script. At least it seems logical.

What do you think?

**Comment by SergeStinckwich** [comment]
Yes @olekscode maybe you should release a stable version with a version number ?

**Comment by SergeStinckwich** [comment]
I was thinking that DataFrame is a part of PolyMath and we should be able to load all PolyMath (including DataFrame) in someway, but also to load DataFrame or other PM packages independantly.
BTW, we need to be able to transform more easily PM structures in DF structures and vice-versa.
